dotnet/src/SemanticKernel/Planning/SKContextPlanningExtensions.cs

```diff
@@ -132,15 +132,17 @@ internal static async Task RememberFunctionsAsync(SKContext context, List<Functi
         foreach (var function in availableFunctions)
         {
             var functionName = function.ToFullyQualifiedName();
-            var key = string.IsNullOrEmpty(function.Description) ? functionName : function.Description;
+            var key = functionName;
+            var description = string.IsNullOrEmpty(function.Description) ? functionName : function.Description;
+            var textToEmbed = function.ToManualString();
```

Comment:
Thanks Abby! I think this is fine for now, although I'm locally considering whether the ManualString is good for this purpose or if just the FunctionName+Description would be better -- I can iterate on this separately, just saying it out loud for those following along. Also considering whether we should include skill name in the key to avoid collisions.